test/test_sparse_csr.py

```diff
@@ -1070,6 +1074,23 @@ def test_coo_csr_conversion(self, device, dtype):
 
             self.assertEqual(csr_sparse.to_dense(), dense)
 
+    @ops(_sparse_csr_ops)
+    def test_sparse_csr_no_error(self, device, dtype, op):
+        samples = op.sample_inputs(device, dtype)
+
+        if len(samples) == 0:
+            self.skipTest("Skipped! No sample inputs!")
+
+        for sample in samples:
+            assert isinstance(sample.input, torch.Tensor)
+            if sample.input.ndim != 2:
+                continue
```

Comment:
With https://github.com/pytorch/pytorch/blob/6c4437118b88c5756b0a192263fb8ed40cfe2800/torch/testing/_internal/common_methods_invocations.py#L1184-L1186, this if-block is redudant.